### DISCUSSION TITLE
Is firebase supported for flutter (windows)

### DISCUSSION THREAD

**Discussion by chetan2914** [firebase/flutterfire#3722] [Q&A] [answered]
I am trying to build a flutter windows application  with firebase authentication and database features.

but iam getting this error, 

[ERROR:flutter/lib/ui/ui_dart_state.cc(177)] Unhandled Exception: MissingPluginException(No implementation found for method Firebase#initializeCore on channel plugins.flutter.io/firebase_core)

main.dart
void main()  async {
  WidgetsFlutterBinding.ensureInitialized();
  await Firebase.initializeApp();
  runApp(MaterialApp(
        debugShowCheckedModeBanner: false,
        initialRoute: '/',
        routes: {
          '/':(context)=>Login(),
        },
      ));
}

pubspec.yaml
dependencies:
  firebase_core: ^0.5.0
  firebase_auth: ^0.18.0+1
  firebase_dart: ^0.7.15+4

**Comment by Salakar** [comment, marked as answer]
Closing in favour of #5557
